python/lsst/db/db.py

```diff
@@ -519,19 +520,22 @@ def _execCommand(self, command, nRowsRet):
         if self._conn is None:
             self.connect()
         with contextlib.closing(self._conn.cursor()) as cursor:
-            self._logger.debug("Executing '%s'." % command)
+            log.debug("Executing '%s'.", command)
```

Comment:
Yes it may. While implemented it, I run into issues with logging, see DM-2242. I am going to workaround it by substituting %s with %%s just for logging